lib/domain/model/contact/contact_status.dart

```diff
@@ -1 +1 @@
-enum ContactStatus { inactive, active }
+enum ContactStatus { inactive, active, external }
```

Comment:
external may be  inactive or active also ?

Reply:
ohh, nice catches. Please use `ContactType`

Reply:
done